homeassistant/components/rfxtrx/manifest.json

```diff
@@ -3,5 +3,6 @@
   "name": "RFXCOM RFXtrx",
   "documentation": "https://www.home-assistant.io/integrations/rfxtrx",
   "requirements": ["pyRFXtrx==0.25"],
-  "codeowners": ["@danielhiversen"]
-}
+  "codeowners": ["@danielhiversen"],
```

Comment:
I think that you quality as a code owner by now :)

Reply:
Damn.... I thought i would sneak some more passed, before i had to shoulder that responsibility ;)